src/fmcore/utils/rate_limit_utils.py

```diff
@@ -0,0 +1,31 @@
+from aiolimiter import AsyncLimiter
+from fmcore.types.config_types import RateLimitConfig
+
+
+class RateLimiterUtils:
+    """
+    Utility class for creating asynchronous rate limiters.
+
+    This class groups functionality related to rate limiting, so you don't have to repeatedly
+    import or write the logic for instantiating rate limiters. It uses the provided RateLimitConfig
+    to create an AsyncLimiter instance.
+    """
+
+    @staticmethod
+    def create_rate_limiter(rate_limit_config: RateLimitConfig) -> AsyncLimiter:
```

Comment:
rename this to create_async_limiter  as it is not returning an interface but a util that returns hard AsyncLimited implementation